dex/bip-id.go

```diff
@@ -619,8 +619,8 @@ var bipIDs = map[uint32]string{
 	// Polygon reserved token range 966000-966999
 	966000: "dextt.polygon",
 	966001: "usdc.polygon",
-	966002: "weth.polygon",
-	966003: "wbtc.polygon",
+	966002: "eth.polygon",
+	966003: "btc.polygon",
```

Comment:
Nah. You're only doing this because you are using the symbol where you should be using the ticker = comventional unit.

Reply:
Conventional Unit is currently "WBTC" and "WETH" also. We are using those for display purposes.. I think people are used to seeing WETH and WBTC when referring to the tokenized version of those assets. Either way I don't think we should be relying on a display name as a source of information.

The reason I removed the "w" over here is that I needed a way to know that "btc" and "wbtc.polygon" refer to the same asset. I think it make sense to have a scheme over here such that the first part of the string allows us to determine what all the equivalent assets are, regardless of the network, and the second part of the string allows us to determine all the assets on a certain network. I'm considering the one part strings to actually be two parts. "eth" actually means "eth.eth". I could make a separate map that does this, but why doesn't it make sense to use this map?

Reply:
> ... I needed a way to know that "btc" and "wbtc.polygon" refer to the same asset

btc and wbtc **absolutely do not** refer to the same asset

> I think it make sense to have a scheme over here such that the first part of the string allows us to determine what all the equivalent assets are, regardless of the network

Again, these are **NOT** the same asset, and I would never support a scheme in which they are treated as equals.

Reply:
Even centralized exchanges do not treat these as the same asset

Reply:
You're right about WBTC.. Binance was allowing withdrawals of BTC to BSC, WBTC on ETH is a separate asset. Not sure what they're doing with BTC on BSC.

What do you think about WETH though? I don't think any exchanges treat WETH and ETH as separate assets.

Reply:
> What do you think about WETH though? I don't think any exchanges treat WETH and ETH as separate assets.

Hmm. That's interesting. WETH != ETH, of course, but I haven't looked at whether exchanges treat them as differently. Regardless, our wallet needs to treat them as differently. They aren't the same.  If you run into an exchange that has WETH enabled for deposit and withdraw, but trades as ETH, you'll just need to code around it in your exchange code. It's going to get even more complicated once we add Arbitrum, which uses ETH as the name of its bridged asset. Ugh.

Reply:
Sounds good, I think it's most exchanges, but it shouldn't be too hard. There will probably just be a lot of special cases  for ETH related things.